src/control/cmd/daos_agent/start.go

```diff
@@ -95,17 +94,23 @@ func (cmd *startCmd) Execute(_ []string) error {
 	// Setup signal handlers so we can block till we get SIGINT or SIGTERM
 	signals := make(chan os.Signal)
 	finish := make(chan struct{})
-	signal.Notify(signals, syscall.SIGINT, syscall.SIGTERM)
 
+	signal.Notify(signals, syscall.SIGINT, syscall.SIGTERM, syscall.SIGPIPE)
 	// Anonymous goroutine to wait on the signals channel and tell the
-	// program to finish when it receives a signal. Since we only notify on
-	// SIGINT and SIGTERM we should only catch this on a kill or ctrl+c
+	// program to finish when it receives a signal. Since we notify on
+	// SIGINT and SIGTERM we should only catch these on a kill or ctrl+c
+	// SIGPIPE is caught and logged to avoid killing the agent.
 	// The syntax looks odd but <- Channel means wait on any input on the
 	// channel.
```

Comment:
If you need to make any changes to this PR, would be good to remove this comment. I think this one comes from very early when we were all first learning Go.

Reply:
Sure, if there is a need to update.